src/content/docs/infrastructure/host-integrations/host-integrations-list/temporal-monitoring-integration.mdx

```diff
@@ -199,6 +221,13 @@ Here is a NRQL query to check the Temporal request latency sum:
 SELECT sum(temp oral_request_latency_sum) FROM Metric WHERE scrapedTargetURL = 'http://<YOUR_DOMAIN>:8000/metrics'
```

Comment:
Is this query correct? I'd check this `sum(temp oral`. For me, it's weird that space... Maybe you could check it with a SME to make sure.